Source/Infrastructure/Read/MongoDbClassMap.cs

```diff
@@ -0,0 +1,23 @@
+using Dolittle.ReadModels;
+using MongoDB.Bson.Serialization;
+
+namespace Infrastructure.Read
+{
+    /// <summary>
+    /// All readmodels in Read should have ClassMaps that derive from this abstract class
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public abstract class MongoDbClassMap<T> : IMongoDbClassMap<T>
+        where T : IReadModel
+    {
+        protected MongoDbClassMap()
+        {
+            if (!BsonClassMap.IsClassMapRegistered(typeof(T)))
+                BsonClassMap.RegisterClassMap<T>(Map);
+        }
```

Comment:
All ReadModels in Read also provide a ClassMap- class that extends MongoDbClassMap.
And when IReadModule is instantiated in Startup, all the ClassMaps are automatically registered on construction